meta_test.go

```diff
@@ -94,3 +95,54 @@ func TestEncodeDecode(t *testing.T) {
 		t.Fatalf("decoded object not equal:\nexpected %#v\nactual   %#v", v, w)
 	}
 }
+
+func benchmarkEncode(n int, t *testing.B) {
+	type test struct {
+		Array []string `json:"array"`
+	}
+	v := &test{}
+	for j := 0; j < n; j++ {
+		v.Array = append(v.Array, "t")
```

Comment:
You could just make this a `n` length string using [`strings.Repeat`](https://golang.org/pkg/strings/#Repeat).